x-pack/test/accessibility/apps/ml.ts

```diff
@@ -13,7 +13,7 @@ export default function ({ getService }: FtrProviderContext) {
   const ml = getService('ml');
 
   // flaky tests, see https://github.com/elastic/kibana/issues/88592
```

Comment:
If test are passing again, it would be good to also remove this skip-related comment.

Reply:
Done in https://github.com/elastic/kibana/pull/89423/commits/f2c1d80f84be7faba054ba9366c54820dbef005d